opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -254,6 +256,23 @@ namespace Opm {
     }
 
 
+    size_t EclipseGrid::activeIndex(size_t i, size_t j, size_t k) const {
+        return activeIndex( getGlobalIndex( i,j,k ));
+    }
+
+
+
+    size_t EclipseGrid::activeIndex(size_t globalIndex) const {
+        assertCellInfo();
```

Comment:
> Can we change this with something like

I would say no - because if you do not have cellinfo you basically have no clue; lack of cellinfo does *not* imply that all cells are active, which should be the case for `return globalIndex;` to be correct. 

When we have implemented a `GridDim` class for the lean grids we can stop this checking.